packages/pharos/src/pages/search/search.scss

```diff
@@ -1,4 +1,4 @@
-@use '@ithaka/pharos/lib/utils/scss/mixins';
+// @use '@ithaka/pharos/lib/utils/scss/mixins';
```

Comment:
this should not be commented out
but scss-loader isn't able to find this file and i'm not sure why 😕